build: comparison of integer expressions of different signedness in api_server_test

```
/home/huangwei/fedb/src/apiserver/api_server_test.cc:681:27: warning: comparison of integer expressions of different signedness: ‘int’ and ‘butil::rapidjson::SizeType’ {aka ‘unsigned int’} [-Wsign-compare]
         for (int i = 0; i < exists_dbs.Size(); ++i) {
```
https://github.com/4paradigm/OpenMLDB/blob/012db063a7f69b55733d3d39db7ed326ad1c2af6/src/apiserver/api_server_test.cc#L681

use decltype

--- a/src/apiserver/api_server_test.cc
+++ b/src/apiserver/api_server_test.cc
@@ -678,7 +678,7 @@ TEST_F(APIServerTest, getDBs) {
         ASSERT_TRUE(document.HasMember("dbs"));
         auto& exists_dbs = document["dbs"];
         ASSERT_TRUE(exists_dbs.IsArray());
-        for (int i = 0; i < exists_dbs.Size(); ++i) {
+        for (decltype(exists_dbs.Size()) i = 0; i < exists_dbs.Size(); ++i) {
             auto db = exists_dbs[i].GetString();
             if (test_dbs.find(db) != test_dbs.end()) {
                 FAIL() << "can't have test db " << db;

--- a/src/cmd/sql_cmd_test.cc
+++ b/src/cmd/sql_cmd_test.cc
@@ -2214,7 +2214,7 @@ TEST_P(DBSDKTest, DeployStatsOnlyCollectDeployProcedure) {
         env.EnableDeployStats();
         absl::SleepFor(absl::Seconds(2));
 
-        for (int i =0; i < 5; ++i) {
+        for (int i = 0; i < 5; ++i) {
             env.CallProcedure();
         }
 

